recipe/meta.yaml

```
@@ -65,7 +80,7 @@ about:
   description: |
    This is the Tango-Controls C++ library (a.k.a. cppTango).
    Tango-Controls is a software toolkit for building control systems.
-  dev_url: https://github.com/tango-controls/cppTango
+  dev_url: https://gitlab.com/tango-controls/cppTango
   doc_url: https://tango-controls.github.io/cppTango-docs
   doc_source_url: https://github.com/tango-controls/cppTango-docs
```

```suggestion
  doc_url: https://tango-controls.gitlab.io/cppTango/
  doc_source_url: https://gitlab.com/tango-controls/cppTango
```
The source of the Doxygen documentation is actually in cppTango code now.
We have a CI/CD job triggered on tags to generate this doc.